src/aiperf/dataset/composer/synthetic_rankings.py

```diff
@@ -64,17 +60,27 @@ def _create_turn(self, num_passages: int) -> Turn:
         """Create a single ranking turn with one synthetic query and multiple synthetic passages."""
         turn = Turn()
 
+        # Generate query with rankings-specific token counts
+        query_tokens = self._query_token_rng.sample_positive_normal_integer(
+            self.config.input.rankings_query_prompt_token_mean,
+            self.config.input.rankings_query_prompt_token_stddev,
+        )
         query_text = self.prompt_generator.generate(
```

Comment:
I wonder if we should break out the base class to support already sampled length?

like if it does:

func a()
- sample size
- generate with size

move the 2nd bullet to its own function that can be called directly?

func a()
- sample size
- call func b(size)

func b(size)
- generate with size

then this code can call  b(size)  directly?